mpcs/src/whir/field_wrapper.rs

```diff
@@ -0,0 +1,600 @@
+use ark_ff::{AdditiveGroup, Field, LegendreSymbol};
```

Comment:
Introduce ark_ff field + wrapper would make slightly large technical burden.

In these case, I would suggest we go from another way to make https://github.com/scroll-tech/whir to use plonky3 field library, so it would be easiler align everything in the future in "production ready" manner